### ISSUE TITLE
Range end handle click area is outside the handle frame

### ISSUE THREAD

**Issue by Professr** [esdete2/react-mirt#11]
### Problem
When I tried this out, the end handle wasn't draggable and didn't have any hover cursor changes.  I tracked it down to the `-webkit-slider-thumb` range element:
![image](https://github.com/user-attachments/assets/b26a521c-e75f-4737-8606-c59ebd080bad)

### Hack-y solution
I was able to get it working by adding this CSS:
```
input[type="range"].mirt__range-handle {
  right: 0;
  width: auto;
}
```
![image](https://github.com/user-attachments/assets/516f9a5f-b01c-4d7e-a2d2-95549a6f16d9)
but I don't know _why_ it wasn't working or if this is a good approach.

### Context / steps to reproduce
I started with a new React project scaffold (`npm create vite my-project`), then cleared out the placeholder stuff from `App.tsx`, `App.css`, and `index.css`.  I threw some code together to fetch a local file and turn it into the `File` object that Mirt expected:
```
import Mirt from 'react-mirt';
import 'react-mirt/dist/css/react-mirt.css';
import { useState } from 'react';
import './App.css'

const audioFilePath = "./samples/snapped-2024-11-15 21:20:04.672239.wav"

function App() {
    const [loading, setLoading] = useState(false)
    const [audioFile, setAudioFile] = useState(null as File | null)

    function loadAudioFile () {
        fetch(audioFilePath).then(async (response) => {
            setAudioFile(await response.blob() as File)
            setLoading(false)
        })
    }

    if( null == audioFile && !loading ) {
        setLoading(true)
        loadAudioFile()
    }

    return (
        <Mirt file={audioFile} />
    )
}

export default App
```

After that it looked and worked great except for the end handle.  Any suggestions for a better or more permanent solution?

**Comment by esdete2** [comment]
First of all, thank you very much for your detailed documentation! ❤️

I was able to identify the problem. The width of the handles (mirt__range-handle) including their padding doesn't correspond to the full width of the sorrounding container (mirt__range-handle-frame), although `width: 100%` is set. As a result, the right handle is outside the component. This is due to a missing `box-sizing: border-box`. This tells the browser that the padding should be included in the calculation of the elements overall width.

I will push a fix, which adds the following style:

```
.mirt,
.mirt * {
  box-sizing: border-box;
}
```

You could also add this for all elements, which is a good idea anyway :)

```
* {
  box-sizing: border-box;
}
```

I'm using the component in a Next.js project, where this style is set by default.

Again, thank you very much and let me know if this fixes the issue!

**Comment by esdete2** [comment]
This should now be fixed in v1.3.6 :)

**Comment by Professr** [comment]
Thank you, it works great! 🥳
